sky/backends/backend_utils.py

```diff
@@ -2557,41 +2556,58 @@ def get_task_demands_dict(task: 'task_lib.Task') -> Dict[str, float]:
     return resources_dict
 
 
-def get_task_resources_str(task: 'task_lib.Task') -> str:
+def get_task_resources_str(task: 'task_lib.Task',
```

Comment:
Are changes here supposed to display in `sky job queue`? See my UX comment. I had an old autostopped controller, then upgraded to the this PR and ran 2 `job launch`.

Reply:
Yes, this is for `sky job queue`. All the existing spot jobs will not have the `(spot)` annotation in the job table, while all newly launched jobs will have them, if it is on an existing controller. Do you think it is necessary to add `(spot)` for those old jobs? I found it may need a few hacky string processing code, which I would rather avoid.